### PR TITLE
Tab reformatting

### PR DESCRIPTION
**Pull request by cm97878**
Moved tabs so they're tied to the node type (currently just 'normal' and 'home'). Fixed the formatting for upgrade buttons and added a tiny transition, moved upgrades to the right spot, spruced up the home panel a bit.

### CODE REVIEW

File: src/stores/mapStore.ts [hunk 1]
```diff
@@ -295,7 +295,7 @@ export const useMapStore = defineStore('mapStuff', () => {
     const totalScouted = computed(() => {
         let val = 0;
         mapNodes.value.forEach(element => {
-            if(element.data.killCount >= element.data.scoutThreshold) {
+            if(!element.data.areaSpecialID && element.data.killCount >= element.data.scoutThreshold) {
                 val++;
             }
```
Comment (Malthusis) [inline src/stores/mapStore.ts:300]: Good to see this case dealt with, was hitting it working on my branch. 👍

File: src/stores/saveStore.ts [hunk 2]
```diff
@@ -17,6 +17,7 @@ export const useSaveStore = defineStore('saveStore', () =>{
         furthestStage: player.furthestStage,
         firstMove: player.firstMove,
         deniedSoul: player.deniedSoul,
+        exploreUnlocked: player.exploreUnlocked,
         currencies: {
```
Comment (Malthusis) [inline src/stores/saveStore.ts:21]: put a comment/todo here for me, I'll be making a more proper fix for this with my shrine work.

File: src/stores/upgradeStore.ts [hunk 3]
```diff
@@ -22,7 +22,13 @@ export const useUpgradeStore = defineStore('upgradeStore', {
...
             }
         */
+
+        //TODO: add this to saving functionality.
        soul: new Map<number, Upgrade>([
+
+        ]),
+        
+        home: new Map<number, Upgrade>([
             [1,{
```
Comment (Malthusis) [inline src/stores/upgradeStore.ts:32]: We'll need to have a talk about what upgrades should be in home/soul, but as is... this completely breaks upgrading as the thing that sets what upgrades are bought is expecting them to be Soul upgrades (and they are labelled as such in their category).

File: src/enums/panels.ts [hunk 4]
```diff
@@ -4,10 +4,14 @@ export enum Panels {
 }
 
 export enum Tab {
-    //World
+    //Nodes
     COMBAT = "combat",
-    AREA_ACTIONS = "areaActions",
-    HOME = "home",
+    INFO = "info",
```
Comment (Malthusis) [inline src/enums/panels.ts:9]: I feel there's a better term then Nodes here... something around Expeditions?

File: src/stores/upgradeStore.ts (hunk 3 above)
Comment (cm97878) [reply src/stores/upgradeStore.ts:32]: Fixed this - i intend soul upgrades to be upgrades specifically utilizing soul mechanically and in terms of flavor. I did find two goofy bugs while checking to see if i could buy them again - you can just keep clicking the upgrades again after you buy em, and if your def > their attack you heal. I'll fix that.

File: src/enums/panels.ts (hunk 4 above)
Comment (cm97878) [reply src/enums/panels.ts:9]: I'll go with Overworld, as the other node types would be special nodes, or the city, or any other non-main-map areas.

**Review by Malthusis — APPROVED** [review]
Looks good now. 👍

**Pull request merged**